themes/default/content/registry/packages/astra/installation-configuration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Astra Installation & Configuration
+title: Astra Setup
```

Comment:
@susanev should this be updated to `Astra Installation & Configuration` in the upstream repo?

Reply:
@ringods it should yes heres my pr from last year for it..
https://github.com/pulumiverse/pulumi-astra/pull/20